magellan-library/src/main/java/com/wealthfront/magellan/lifecycle/ActivityLifecycleAdapter.kt

```diff
@@ -46,10 +46,11 @@ internal class ActivityLifecycleAdapter(
   }
 }
 
-fun ComponentActivity.setContentScreen(
-  navigable: Navigable,
-  @LayoutRes root: Int = R.layout.magellan_root
-) {
+fun ComponentActivity.setContentScreen(navigable: Navigable, @LayoutRes root: Int = R.layout.magellan_root) {
   setContentView(root)
   lifecycle.addObserver(ActivityLifecycleAdapter(navigable, this))
 }
+
+fun ComponentActivity.setExpedition(navigable: Navigable) {
+  lifecycle.addObserver(ActivityLifecycleAdapter(navigable, this))
```

Comment:
This method is necessary for us where we inflate the xml before the navigable is available through dependency injection

Reply:
Wait, when does this happen?

Reply:
This is gonna be mostly for migration purposes and we will mark this as opt in...